config/params/config.go

```diff
@@ -193,11 +196,55 @@ type BeaconChainConfig struct {
 // InitializeForkSchedule initializes the schedules forks baked into the config.
 func (b *BeaconChainConfig) InitializeForkSchedule() {
 	// Reset Fork Version Schedule.
-	b.ForkVersionSchedule = map[[4]byte]types.Epoch{}
+	b.ForkVersionSchedule = map[[fieldparams.VersionLength]byte]types.Epoch{}
 	// Set Genesis fork data.
 	b.ForkVersionSchedule[bytesutil.ToBytes4(b.GenesisForkVersion)] = b.GenesisEpoch
 	// Set Altair fork data.
 	b.ForkVersionSchedule[bytesutil.ToBytes4(b.AltairForkVersion)] = b.AltairForkEpoch
 	// Set Bellatrix fork data.
 	b.ForkVersionSchedule[bytesutil.ToBytes4(b.BellatrixForkVersion)] = b.BellatrixForkEpoch
 }
+
+// ForkScheduleEntry is a Version+Epoch tuple for sorted storage in an OrderedForkSchedule
```

Comment:
This seems a lot better suited for this package:
https://github.com/prysmaticlabs/prysm/blob/develop/network/forks

We handle the versioning of forks and ordering it over there

Reply:
What do you think about moving these types/methods to that package, but instead of:
`func (b *BeaconChainConfig) OrderedForkSchedule() OrderedForkSchedule {`
define:
`func NewOrderedForkSchedule(b *BeaconChainConfig) OrderedForkSchedule {`
with the same internal logic?

Reply:
Sounds good to me :+1:

Reply:
moved, PTAL